core/domain/suggestion_services.py

```diff
@@ -412,6 +412,25 @@ def accept_suggestion(
         _update_user_proficiency(user_proficiency)
 
 
+def update_translation_suggestion(suggestion_id, translation_html):
+    """Updates the change.translation_html of a suggestion with the
+        given suggestion_id.
+
+    Args:
+        suggestion_id: str. The id of the suggestion to be accepted.
+        translation_html: str. The new translation html.
+
+    Returns:
+        Suggestion. The updated suggestion.
```

Comment:
Why this function returns a suggestion? I don't see any usage of that return value anywhere, am I missing something?

Reply:
Removed it.